src/Icicle/Source/Query/Exp.hs

```diff
@@ -93,7 +93,7 @@ data Fun
  | DaysBetween
  | DaysEpoch
  | Seq
- deriving (Show, Eq, Ord)
+ deriving (Show, Eq, Ord, Enum, Bounded)
```

Comment:
can you make a new function here, ``listOfAllPrims :: [Prim]``? I don't mind the enum/bounded trick but I'd rather it be very localised. that way if we decide to add more interesting, non-enumerable prims, we only need to modify this file